evaluation/README.md

```diff
@@ -1 +1,21 @@
-# Evaluate your app
+# Evaluate your AI app
+
+## Steps to run evaluation
+
+1. Install required libraries with the following command under project root:
+   ```bash
+   pip install -r ai-resource/evaluation/requirements.txt
+   ```
+1. Import your AI app and wrap it into evaluation target (a function named `target_app` in the script). 
+1. If AI-assisted evaluator is selected, set language model's api key in the `.env` file or environment variable.
```

Comment:
Do we need to introduce the sdk and built-in evaluator a little bit?

Reply:
Updated.